README.md

```diff
@@ -123,6 +123,11 @@ Similarly, to delete without confirmation:
 ```bash
 ./dtm -y delete -f YOUR_CONFIG_FILE.yaml
 ```
+To delete everything defined in the config, regardless of the state: 
+
+```bash
+./dtm delete -r -f YOUR_CONFIG_FILE.yaml
```

Comment:
I think we need a better parameter here.

"-f" is taken by "--from-file" but "-r" doesn't relate to "force" that much.

What do you think?

I suggest --force, no short form, because "-f" is taken, and because force delete is dangerous, so it doesn't hurt to let the user type more to make sure this is really what they want.